src/components/template/TemplateTopicEditor.tsx

```diff
@@ -187,60 +213,53 @@ export default function QuestionaryEditorTopic(props: {
       {({ isSubmitting, handleChange, values }) => (
         <Form>
           {isEditMode ? (
-            <Grid container alignItems="center">
-              <Grid item xs={8}>
-                <Field
-                  name="title"
-                  id="title"
-                  label="Topic Title"
-                  type="text"
-                  component={TextField}
-                  className={classes.title}
-                  value={values.title}
-                  onChange={handleChange}
-                  margin="normal"
-                  fullWidth
-                  data-cy="topic-title-input"
-                  required
-                />
-              </Grid>
-              <Grid item xs={2}>
-                <Button
-                  disabled={isSubmitting}
-                  variant="text"
-                  size="small"
-                  color="primary" // secondary + text variant is bad for contrast
-                  onClick={() => setIsEditMode(false)}
-                  className={classes.button}
-                  data-cy="topic-title-cancel-update"
-                >
-                  Cancel
-                </Button>
-              </Grid>
-              <Grid item xs={2}>
-                <Button
-                  disabled={isSubmitting}
-                  type="submit"
-                  size="small"
-                  variant="contained"
-                  color="primary"
-                  className={classes.button}
-                  data-cy="topic-title-update"
-                >
-                  Update
-                </Button>
-              </Grid>
-            </Grid>
-          ) : (
-            <div>
-              <span data-cy="topic-title">{values.title}</span>{' '}
-              <EditIcon
-                fontSize="small"
-                data-cy="topic-title-edit"
-                onClick={() => {
-                  setIsEditMode(true);
+            <div style={{ display: 'flex' }}>
+              <Field
```

Comment:
Changes look good! Had to look over with the [Wave tool](https://wave.webaim.org/) for any accessibility concerns and it showed a missing form label when editing a topic name would you be able to add one with aria-label to this field maybe?

Reply:
Good point I added the label for the screen readers